Makefile

```diff
@@ -154,7 +154,7 @@ test: prepare-test-context  ## - Run all tests
 
 .PHONY: test-release
 test-release:  ## - Check that all release binaries are created
-	./.ci/scripts/test-release.sh $(DEFAULT_VERSION)
+	./.buildkite/scripts/test-release.sh $(DEFAULT_VERSION)
```

Comment:
Should the `test-release.sh` file be moved to `.buildkite`?

Reply:
Yes 🤦🏼